### ISSUE TITLE
Implement persistent storage

### ISSUE THREAD

**Issue by frejfrej** [kungfuwushu/backend#3]
Initial implement will not focus on persistent storage.

This issue must focus on providing a flexible way for storing data, with impacts on retrieving it. That includes documentation and tests.

**Comment by jsorette** [comment]
A simple way to persist h2 database by storing it on a file : 
```
spring.datasource.url=jdbc:h2:file:~/kungfuclub-db;DB_CLOSE_ON_EXIT=FALSE;AUTO_RECONNECT=TRUE
spring.datasource.driver-class-name=org.h2.Driver
spring.jpa.hibernate.ddl-auto=update
```

**Comment by frejfrej** [comment]
If we go with this, we might also want a "fixture" (a fil -standalone java program in our case- that creates default testing data on load when the database does not exist)

**Comment by jsorette** [comment]
Absolutely, we may add `import.sql` file to the classpath which allows to automatically load data when `spring.jpa.hibernate.ddl-auto` is either `create-drop` or `create` : 
```
// import.sql
INSERT INTO profile (id, firstname, lastname) VALUES (1, 'Prenom', 'Nom');
...
```
See [Spring doc : Initialize a Database Using Hibernate](https://docs.spring.io/spring-boot/docs/current/reference/html/howto-database-initialization.html#howto-initialize-a-database-using-hibernate)
